R/calcHistorical.R

```diff
@@ -20,6 +20,10 @@ calcHistorical <- function() {
   pe_weo <- collapseNames(pe_weo)
   pe_weo <- add_dimension(pe_weo, dim = 3.1, add = "model", nm = "IEA_WEO")
 
+  # IEA EB direct sums
+  iea_eb_direct_sums <- calcOutput('IEA_EB_directSum') |>
+    add_dimension(dim = 3.1, add = 'model', nm = 'IEA-EB-directSum')
```

Comment:
you can use the parameter `writeArgs = list(scenario = "historical", model = "IEA-EB-directSum")` in calcOutput for this